test/README.md

```diff
@@ -8,6 +8,19 @@ machine state. The tests are deterministic, and each test is run in a virtual
 machine with fresh state, ensuring that tests that modify system state are
 easily repeatable.
 
+Installing the Test environnement
+---------------------------------
+
+### Debian-based distributions
+
+- Install the requirements
+
+      # apt install ruby-dev vagrant
```

Comment:
1. You mean replace

```
- Install the requirements

      # apt install ruby-dev vagrant
```

with

````no-highlight
- Install the requirements

```shell
# apt install ruby-dev vagrant
```
````

, right?

2. Well, I'm used to:
- [use `#` (over `$`)](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt) to tell the command needs to be executed as `root`:
> On many Unix and derivative systems, the prompt commonly ends in **`$` or `%` if the user is a normal user, but in `#` if the user is a superuser** ("root" in Unix terminology).

- run `apt` over `apt-get` as it seems to be the [new standard](https://www.debian.org/doc/manuals/debian-handbook/sect.apt-get.en.html#sect.apt-get) for Debian-based distributions:

> APT is a vast project, whose original plans included a graphical interface. It is based on a library which contains the core application, and `apt-get` is the first front end — command-line based — which was developed within the project. **`apt` is a second command-line based front end provided by APT which overcomes some design mistakes of `apt-get`**.